src/translation/i18n.json

```diff
@@ -81,7 +83,8 @@
     "system": {
       "aeropress": "AeroPress",
       "moka": "Kawiarka",
-      "v60": "V60"
+      "v60": "V60",
+      "frenchPress": ""
```

Comment:
You can just put "FrenchPress" in to translations where you don't know the exact word. I think French Press is an international name